app/assets/stylesheets/components/_header.scss

```diff
@@ -1,86 +1,66 @@
-.usa-site-navbar {
+.header {
   background: $primary-darkest;
-  padding-bottom: 10px;
-  padding-top: 10px;
 }
 
-.usa-site-navbar a {
+.header-wrap {
+  padding-top: 1.875rem;
+}
+
+.header-title {
+  float: left;
+  padding-bottom: 1rem;
+}
+
+.header-side {
+  padding-top: 1rem;
   color: $color-white;
-  font-weight: $font-normal;
-  margin-left: 14px;
-  padding-left: $site-margins;
-  padding-right: $site-margins;
-  position: relative;
-  vertical-align: top;
 
-  &.header-home-link {
-    display: table-cell;
-    margin-left: 0;
-    margin-right: 0;
-    padding-left: 0;
-    padding-right: 0;
-    text-decoration: none;
+  a {
+    color: $color-white;
+  }
+
+  @include media($medium-screen) {
+    ul {
+      float: right;
+      margin: 0;
+    }
 
-    &:hover,
-    &:focus {
-      text-decoration: none;
+    li {
+      float: right;
+      padding-left: 1em;
```

Comment:
Does this use em so that the indenting will stay relatively constant even if the user resizes the text? Just trying to deal with my own lack of CSS knowledge